zeroconf/__init__.py

```diff
@@ -1090,17 +1092,17 @@ class DNSCache:
     """A cache of DNS entries"""
 
     def __init__(self):
-        self.cache = {}  # type: Dict[str, List[DNSEntry]]
+        self._cache = {}  # type: Dict[str, List[DNSEntry]]
```

Comment:
This change seems out of scope of this pull request – is it necessary for some reason?

Reply:
It's not necessary, but I'm used to the boyscout rule (eg; leave the code you touched better than you left it).
In this case, the DNS cache uses a dict to do it's internal work, but it's not meant as an interface. As such, it should be protected / private. 
Anyway, I could also do that in a seperate pull request.

Reply:
That's fair, although it's not clear that it's actually better this way (it's questionable and I'd not bother with it) and I don't think this principle applies to programming and pull requests, as unrelated improvements create noise.

Reply:
I'll remove it from this pull request. We can have a more in depth discussion as to why this is better in the other pull request. I do have a bunch of arguments as to why this is better.